archive/k/kotlin/README.md

```diff
@@ -9,6 +9,7 @@ Welcome to Sample Programs in Kotlin!
   - Solution borrowed from @toturkmen via the [baklava repo][1]
 - [Fizz Buzz in Kotlin][7]
 - [Fibonacci Sequence in Kotlin][8]
+- [Reverse String in Kotlin][]
```

Comment:
Can you create an article issue for this solution like #566 and link it here.

Reply:
Thanks! I can do that.